values/jobs/autoconfig.gotmpl

```diff
@@ -0,0 +1,27 @@
+{{- $v := .Values }}
+{{- $c := $v.charts }}
+{{- $k := $c.keycloak }}
+type: Job
+name: autoconfig
+
+image:
+  repository: docker.io/otomi/otomi-e2e
+  tag: autoconfig
+  pullPolicy: Always
+
+script: | 
+    echo "AUTOCONFIG TASK: Creating Keycloak user"
+    yarn cypress run --record false --env BASE_URL={{$v.cluster.domainSuffix}} --spec 'cypress/integration/otomi-applications/10-keycloak.feature'
+    echo "AUTOCONFIG TASK: Actuvate drone"
```

Comment:
Actuvate -> Activate
Haven't spellcheck complained about it?

Reply:
Done